src/test/java/com/snack/news/service/NewsServiceTest.java

```diff
@@ -17,8 +17,7 @@
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyLong;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'org.mockito.Mockito'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)